packages/effects-core/src/math/value-getters/value-getter-map.ts

```
@@ -0,0 +1,93 @@
+import { colorToArr, isFunction } from '../../utils';
+import { BezierCurve, BezierCurvePath, BezierCurveQuat, GradientValue, LinearValue, LineSegments, PathSegments, RandomSetValue, RandomValue, RandomVectorValue, StaticValue } from './value-getter';
+import { Vector3 } from '@galacean/effects-math/es/core/vector3';
+import { Quaternion } from '@galacean/effects-math/es/core/quaternion';
+import { ColorCurve } from './color-curve';
+import { Vector4Curve } from './vector4-curve';
+import { ValueGetter } from './value-getter';
+import { HELP_LINK } from '../../constants';
+
+const map: Record<any, any> = {
+  [spec.ValueType.RANDOM] (props: number[][]) {
+    if (props[0] instanceof Array) {
```

Biome

[error] Use Array.isArray() instead of instanceof Array.

instanceof Array returns false for array-like objects and arrays from other execution contexts.
Unsafe fix: Use Array.isArray() instead.


(lint/suspicious/useIsArray)